test/integration/host_js_test.rb

```diff
@@ -521,35 +521,37 @@ class HostJSTest < IntegrationTestWithJavascript
 
     # At this point in time this only tests the UI. Since Fog mocks all requests the Hard drive change is ignored as it
     # will always return the mocked values. Not a total waste however as the UI is still tested with this.
-    test 'enables changing vm disk size' do
-      Fog.mock!
-
-      organization = Organization.find_by_name('Organization 1')
-      location = Location.find_by_name('Location 1')
-      uuid = '5032c8a5-9c5e-ba7a-3804-832a03e16381'
-      compute_resource = FactoryBot.create(:compute_resource, :vmware, organizations: [organization], locations: [location], uuid: 'Solutions')
-      compute_resource = ComputeResource.find_by_id(compute_resource.id)
+    context 'with vmware' do
+      setup do
+        Fog.mock!
```

Comment:
There is a general `Fog.mock!` in before. Is that not used?

Reply:
good point. I'll try removing these.

Reply:
hmm, now i'm seeing the same error locally even with this code, looks like i need to dig deeper.